static/js/components/tiles/histogram_tile.tsx

```diff
@@ -0,0 +1,190 @@
+}
+
+interface HistogramData {
+  dataPoints: DataPoint[];
+  sources: Set<string>;
+}
+
+/**
+ * Main histogram tile component
+ */
+export function HistogramTile(props: HistogramTilePropType): JSX.Element {
+  const svgContainer = useRef(null);
+  const [rawData, setRawData] = useState<SeriesApiResponse | undefined>(null);
+  const [histogramData, setHistogramData] = useState<HistogramData | undefined>(
+    null
+  );
+
+  useEffect(() => {
+    fetchData(props, setRawData);
+  }, [props]);
+
+  useEffect(() => {
+    if (rawData) {
+      processData(props, rawData, setHistogramData);
+    }
+  }, [props, rawData]);
+
+  useEffect(() => {
+    if (histogramData) {
+      draw(props, histogramData.dataPoints);
+    }
+  }, [props, histogramData]);
+
+  if (!histogramData) {
+    return null;
+  }
+  const rs: ReplacementStrings = {
+    place: props.place.name,
+    date: "",
+  };
+  return (
+    <ChartTileContainer
+      title={props.title}
+      sources={histogramData.sources}
+      replacementStrings={rs}
+      className="histogram-chart"
+    >
+      <div id={props.id} className="svg-container" ref={svgContainer}></div>
+    </ChartTileContainer>
+  );
+}
+
+/**
+ * Gets the series of observations to plot
+ */
+function fetchData(
+  props: HistogramTilePropType,
+  setRawData: (data: SeriesApiResponse) => void
+): void {
+  const statVars = [];
+  for (const spec of props.statVarSpec) {
+    statVars.push(spec.statVar);
+    if (spec.denom) {
+      statVars.push(spec.denom);
+    }
+  }
+  axios
+    .get("/api/observations/series", {
+      // Fetch both numerator stat vars and denominator stat vars
+      params: {
+        variables: statVars,
+        entities: [props.place.dcid],
+      },
+      paramsSerializer: stringifyFn,
+    })
+    .then((resp) => {
+      setRawData(resp.data);
+    })
+    .catch((error) => {
+      console.log(error);
+      setRawData(null);
+    });
+}
+
+/**
+ * Process and set histogram data
+ */
+function processData(
+  props: HistogramTilePropType,
+  rawData: SeriesApiResponse,
+  setHistogramData: (data: HistogramData) => void
+): void {
+  const histogramData = rawToChart(rawData, props);
+  setHistogramData(histogramData);
+}
+
+/**
+ * Renders the histogram
+ */
+function draw(props: HistogramTilePropType, histogramData: DataPoint[]): void {
+  const elem = document.getElementById(props.id);
+  // TODO: Remove all cases of setting innerHTML directly.
+  elem.innerHTML = "";
+  drawHistogram(
+    props.id,
+    elem.offsetWidth,
+    CHART_HEIGHT,
+    histogramData,
+    props.statVarSpec[0].unit
+  );
+}
+
+/**
+ * Performs conversion of raw series data to HistogramData object
+ */
+function rawToChart(
+  rawData: SeriesApiResponse,
+  props: HistogramTilePropType
+): HistogramData {
+  // (TODO): We assume the index of numerator and denominator matches.
+  // This is brittle and should be updated in the protobuf that binds both
+  // together.
+  const raw = _.cloneDeep(rawData);
```

Comment:
curious why you're creating a clone of the rawData? It looks like `raw` just gets read in the subsequent code and not modified, so why not just read from rawData?

Reply:
The other _tile.tsx files seem to do this, so I copied their behavior. Removed the cloneDeep.